middleware/validation.js

```diff
@@ -84,6 +85,38 @@ export const validateLogin = [
     .withMessage('Please provide a valid password.'),
 ];
 
+export const validateUser = async (req, res, next) => {
+  const {
+    user,
+    params: { slug }
+  } = req;
+  try {
+    const article = await Article.findOne({
+      where: {
+        slug
+      }
+    });
+    if (!article) {
+      return res.status(404).json({
+        success: false,
+        errors: ['Article not found']
+      });
+    }
+    const checkUser = article.userId === user.id;
```

Comment:
@Dubby20 there's no need to save this to a variable if you're just using it below.

Reply:
Okay I've removed the variable